api/phone.go

```diff
@@ -47,7 +48,9 @@ func (a *API) sendPhoneConfirmation(tx *storage.Connection, ctx context.Context,
 		return err
 	}
 
-	if serr := smsProvider.SendSms(phone, user.ConfirmationToken); serr != nil {
+	message := fmt.Sprintf("Your OTP for %s is %v.", config.SiteURL, user.ConfirmationToken)
```

Comment:
really pedantic but can you remove the full stop, it might prevent some phones from auto parsing the pin

Reply:
oh whoops i did not know that haha ok removing now